dashboard/app/controllers/registrations_controller.rb

```diff
@@ -295,11 +295,11 @@ def set_student_information
     student_information = {}
 
     student_information[:age] = params[:user][:age] if current_user.age.blank?
-    student_information[:us_state] = params[:user][:us_state] unless current_user.user_provided_us_state
+    us_state_param = params[:user][:us_state]
+    student_information[:us_state] = us_state_param if us_state_param && !current_user.user_provided_us_state
```

Comment:
Nit: could there ever be a case where `us_state_param` is an empty string rather than `nil`? If so, I think using `.present?` would handle this, I.e. `us_state_param.present?`

Reply:
Yeah that's a good catch, I'll change the check.